plugins/modules/aci_l3out_logical_node.py

```diff
@@ -294,6 +330,9 @@ def main():
 
     child_classes = ["l3extLoopBackIfP"]
 
+    if mpls_transport_loopback_address is not None:
+        child_classes.append("mplsNodeSidP")
```

Comment:
is it mandatory to have a loopback address to add a SID to the node? because it seems like the mplsNodeSidP have multiple dn:
[0]: uni/tn-{name}/out-{name}/lnodep-{name}/rsnodeL3OutAtt-[{tDn}]/lbp-[{addr}]/nodesidp-{sidoffset}
[1]: uni/tn-{name}/out-{name}/lnodep-{name}/rsnodeL3OutAtt-[{tDn}]/nodesidp-{sidoffset}